maint-lib/stagelib/envglobals.py

```diff
@@ -15,6 +15,7 @@
 # Allow lines to extend beyond 99 char limit by 6 chars to support output formatting to 100 cols
 REQUIRED_ENV_VARS = OrderedDict([
     ('CEPH_VERSION',      'Ceph named version being built (e.g., luminous, mimic)'),
```

Comment:
The doc for `CEPH_VERSION` needs to change to make note that it is allowed to be `luminous` or `luminous-1.2.3-1` point release. I think we **definitely** want to avoid changing what an input variable is set to during staging, so we should use `CEPH_VERSION` to compute 2 new vars: `CEPH_VERSION_NAME` (which replaces the old `CEPH_VERSION`) and `CEPH_POINT_RELEASE`.

Reply:
I forgot that one. I updated the documentation.
CEPH_VERSION will always be luminous and CEPH_POINT_RELEASE will be empty or set.
So for a user point of view, the semantic is always the same.
I do agree about the confusion that it could create since we rewrite CEPH_VERSION but that reduce the number of variables to handle/support.